torch/lib/c10d/frontend.cpp

```diff
@@ -918,4 +920,428 @@ void DistributedC10d::registerProcessGroupName(const c10::intrusive_ptr<ProcessG
   pg_names_[process_group] = name;
 }
 
+void initCustomClassBindings() {
```

Comment:
I was struggling to get the linker not throw out these global static initializers on all platforms, so I gave up and made the initialization explicit, which seems like better practice anyway. Currently it's being initialized when we initialize the rest of the Python bindings (e.g. from `libtorch_python`), but since the actual binding code is in `libc10d` we can initialize it from other places (like `libtorch_deploy`).